src/components/RuleBuilder/Group.jsx

```diff
@@ -36,6 +36,36 @@ export const Group = ({
     [andLabel, orLabel]
   )
 
+  const getProcessedCombinedRuleOptions = (ruleIndex, currentType) =>
+    Object.fromEntries(
```

Comment:
ma doare capul la ce e aici .. filter in enum in filter in reduce in alt filter 

doamne ajuta-l pe urmatorul care trebuie sa schimbe aici ceva